Full Screen Issue solved

aritroCoder opened the pull request:
First time pressing fullscreen makes the window fullscreen while second time press exits fullscreen.

RAJAGOPALAN-GANGADHARAN reviewed (commented):
Hello @aritroCoder , thanks for this PR. Can you please remove package-lock.json out of this commit?

RAJAGOPALAN-GANGADHARAN commented on src/desktop/notification/notification.js:
```diff
@@ -1,27 +1,30 @@
+            this.props.parent_class.setState({ isFull: false });
+        else{
+            this.props.parent_class.setState({ isFull: true });
+            document.webkitExitFullscreen();
```
Why is there a webkit specific function? What about chrome?

aritroCoder replied:
I have tested it in chrome its working there

aritroCoder replied:
Its working in chrome, edge, opera , brave so I think it is no problem

RAJAGOPALAN-GANGADHARAN replied:
Actually can you tell me what document.webkitExitFullScreen is doing? IIRC The isFull flag will signal FullScreen React component to do that for you. Can you remove that and test if it works? Thanks!

aritroCoder replied:
I had tried what you had told before but it was not working for me. The screen was getting into fullscreen mode but it was not reverting back. I have checked the official docs of react-fullscreen myself but I did not found any way of reverting back from fullscreen. isFull state seems to be working fine only for getting into fullscreen. So I used the DOM Document object to manipulate the fullscreen state. I updated the props just to make sure  it can get back into fullscreen again. I think this is the only way to solve this issue while having minimal impact on the existing codebase.

RAJAGOPALAN-GANGADHARAN commented on src/desktop/desktop.js:
```diff
@@ -37,6 +37,10 @@ export default class Desktop extends Component
     {
         this.setState({ isFull: true });
     }
+    stopFullScreen()
```
Could you tell me what does this function do? If you feel fullScreen() function written by me is also not used then probably remove it. As I think the fullscreen is being handled within child IIRC

aritroCoder replied:
The fullscreen code you wrote was not working, the state change was directly accepted and used by the react fullscreen package. I have cleaned up the code by removing those functions

RAJAGOPALAN-GANGADHARAN commented:
It's a better idea to make a separate PR for linting changes as it distracts from the main changes.

aritroCoder commented:
> Hello @aritroCoder , thanks for this PR. Can you please remove package-lock.json out of this commit?

I have added package-lock.json in gitignore and removed from repo

RAJAGOPALAN-GANGADHARAN commented:
Rebased and fixed commit #49 , thanks for the work!
